homeassistant/components/tuya/switch.py

```diff
@@ -514,6 +514,28 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Curtain
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf46o5mtfyc
+    "cl": (
+        SwitchEntityDescription(
+            key=DPCode.CONTROL_BACK,
+            name="Reverse Operation",
+            icon="mdi:restore",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SwitchEntityDescription(
+            key=DPCode.OPPOSITE,
+            name="Reverse Setting",
+            icon="mdi:restore",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SwitchEntityDescription(
+            key=DPCode.AUTO_POWER,
+            name="Automatically Start",
```

Comment:
What does this do?

Reply:
I don't know, the only information available in the documentation is the description.

I included so that if there is this control, the user has it. Better take it off?

Reply:
Well if it is not clear what it does, we won't be able to ensure it is represented or described correctly either :)

Reply:
Removed!